### PR TITLE
198 add val data to training

### LINKED ISSUE #198: Incorporate validation data into train set, evaluate on test
We need to incorporate what has been the validation data into the train set and evaluate on the test set. Once this is done, we will not be making any more modifications to the models.

### PR DESCRIPTION
**Pull request by jsadler2**
I moved the old validation data into the training set and I moved the old "test" set into the validation set. ⚠️  This PR will constitute the final results for the paper since we would no longer have a test set.

closes #198

### CODE REVIEW

**Review by galengorski — APPROVED** [review]
This looks good to me, although I did not run it myself. Are there still temporal hold outs? Or do we only have spatial hold outs now?

**Comment by jsadler2** [comment]
There are temporal (2015-2021) and spatial holdouts (01475530, 01475548)

File: _targets.R [hunk 1]
```diff
@@ -84,16 +84,13 @@ dummy_date <- "2022-06-15"
...
 # Define train/val/test dates
 train_start_date <- '1980-01-01'
-train_end_date <- '2014-10-01'
-val_start_date <- '2014-10-01'
-val_end_date <- '2015-10-01'
-test_start_date <- '2015-10-01'
-test_end_date <- '2022-10-01'
+train_end_date <- '2015-10-01'
+val_start_date <- '2015-10-01'
+val_end_date <- '2021-10-01'
```
Comment (lekoenig) [inline _targets.R:93]: @jsadler2 did you mean to stop the validation time period in 2021? The previous `test_end_date` went out to `'2022-10-01'`. I'm also asking because I'm wondering if I should extend the NWIS pull date beyond the `latest_date` which is currently set to `"2021-12-31"`.  

```suggestion
val_end_date <- '2022-10-01'
```

Comment (jsadler2) [reply _targets.R:93]: See #135 for the resolution of this

**Comment by jsadler2** [comment]
closed this b/c I want merge with #204
